pages/content/amp-dev/documentation/courses/beginning-course/course-introduction@pt_BR.md

```diff
@@ -0,0 +1,106 @@
+
+A caixa vermelha acima indica o editor online onde você digitará HTML e CSS. A caixa verde indica o botão que o levará para a versão online da página que você está criando. A caixa amarela é o botão que permite criar uma cópia deste projeto e editá-lo. A caixa azul indica os arquivos que você tem disponíveis. Na pasta de assets, você pode encontrar suas imagens.
+
+Ao longo desses cursos, você precisará de várias imagens para completar os exercícios. Todas as imagens de que você precisa para concluir esses cursos estão contidas em nossos projetos Glitch. Para visualizar as imagens em seu projeto, clique na entrada de ativos na lista de arquivos no lado esquerdo do editor de Glitch. Para obter o link de qualquer imagem, selecione a imagem na lista de ativos no lado direito. Clique no botão “copiar” ao lado do URL no pop-up que aparece. Você pode usar esse link em qualquer lugar em que uma imagem seja necessária.
+
+{{ image('/static/img/courses/beginner/image8.png', 843, 468,  align='center', caption='A visualização de ativos no Glitch') }}
+
+{{ image('/static/img/courses/beginner/image12.png', 371, 354,  align='center', caption='O pop-up de detalhes (incluindo URL) para uma imagem na coleção de ativos') }}
+
+Neste curso, começaremos com uma página HTML básica. Criamos um projeto vazio no Glitch contendo algumas imagens, o código do servidor que você precisará mais tarde e um arquivo index.html com um título e uma única imagem.
+
+Abra <a href="https://glitch.com/edit/#!/nosy-leech" target="_blank">esse projeto</a> para começar. Clique no botão “Remix This” no lado superior direito para criar um novo projeto que você pode editar. Você pode continuar a usar este editor para este e futuros cursos. Cada curso futuro também lhe dará a oportunidade de começar com uma versão de referência da solução até aquele ponto.
+
+Você não precisa usar o Glitch para concluir esses treinamentos. No entanto, parte do código necessário para completar os exercícios está contido apenas nos exemplos de Glitch. Se desejar usar outro editor, ainda pode ser necessário usar os exemplos de Glitch para encontrar este código.
+
+## Configurando o AMP Validator <a name="setting-up-the-amp-validator"></a>
+
+Para detectar erros em nossas páginas AMP, temos uma ferramenta valiosa ao nosso alcance: o validador de AMP. Escrever páginas AMP válidas é a chave para acessar todos os benefícios do framework. O validador de AMP pode ser acessado de duas maneiras: por meio de uma extensão do Chrome ou adicionando um parâmetro ao nosso URL para que nossa página AMP use o validador integrado. Para os fins deste curso, sugerimos que você use a extensão do Chrome, pois é mais fácil de usar e acessar enquanto você constrói seu site.
+
+- Para instalar a extensão do Chrome, visite o link [aqui](https://chrome.google.com/webstore/detail/amp-validator/nmoffdblmcmgeicmolmhobpoocbbmknc/related?hl=en).
```

Comment:
@alvarocamillont you can reference the portuguese version on this link https://chrome.google.com/webstore/detail/amp-validator/nmoffdblmcmgeicmolmhobpoocbbmknc/related?hl=pt_br

Reply:
Done